wix-formbuilder/$w/WixForms/guides/Migrate from WixForms.md

```diff
@@ -0,0 +1,38 @@
+# Migrate from WixForms to WixFormsV2
+As of April 2025, [WixForms](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms/introduction) is deprecated and any new forms created will only work with [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction). You can access the functionality of WixForms using [WixFormsV2](https://dev.wix.com/docs/velo/velo-only-apis/$w/wix-forms-v2/introduction).
+
+Old forms will only work with the old APIs, WixForms, WixFormSubmitEvent, WixFormSubmittedErrorEvent, and WixFormSubmittedEvent. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, there's a tab for each. There is no easy way to distinguish in the editor, except that their menus are different.
```

Comment:
```suggestion
Old forms will only work with the old APIs: `WixForms`, `WixFormSubmitEvent`, `WixFormSubmittedErrorEvent`, and `WixFormSubmittedEvent`. You can see which forms are old forms and which are new forms by looking at the forms and submissions page in a site dashboard, where there's a tab for each. 
```
Please check that everything that should be in code font is in code font throughout (e.g. `WixForms`, `WixFormsV2`)

Also, why not link to the old APIs since you are mentioning them? 

(Same comments apply for the other places this paragraph appears.)